PSKoans/Public/Measure-Karma.ps1

```diff
@@ -43,7 +43,13 @@ function Measure-Karma {
 
         [Parameter(Mandatory, ParameterSetName = "Reset")]
         [switch]
-        $Reset
+        $Reset,
+
+        #Help is not listed since it attaches to Karma Default actions
+        [Parameter()]
+        [Alias('Dojo', 'Spar')]
+        [switch]
+        $ShowHelp
```

Comment:
This should probably be named simply `-Help` and the `-Dojo` alias seems a bit forced. I'm a fan of `-Spar` though. Maybe something more intuitive like `-Hint` would be a good secondary alias?